Resources/Locale/en-US/_RMC14/xeno/xeno-nest.ftl

```diff
@@ -1,12 +1,13 @@
-cm-xeno-nest-securing-self = You secrete a thick, vile resin, securing {$target} into the alien nest!
+cm-xeno-nest-securing-self = We secrete a thick, vile resin, securing {$target} into the alien nest!
 cm-xeno-nest-securing-target = {$user} secretes a thick, vile resin, securing you into the alien nest!
 cm-xeno-nest-securing-observer = {$user} secretes a thick, vile resin, securing {$target} into the alien nest!
 
-cm-xeno-nest-pin-self = You pin {$target} into the alien nest, preparing the securing resin.
+cm-xeno-nest-pin-self = We pin {$target} into the alien nest, preparing the securing resin.
 cm-xeno-nest-pin-target = {$user} pins you into the alien nest, preparing the securing resin.
 cm-xeno-nest-pin-observer = {$user} pins {$target} into the alien nest, preparing the securing resin.
 
 cm-xeno-nest-failed = {$target} can't be put into a nest!
 cm-xeno-nest-failed-target-resisting = {$target} is resisting, ground them!
-cm-xeno-nest-failed-cant-there = You can't create a nest there!
+cm-xeno-nest-failed-cant-there = We can't create a nest there!
 cm-xeno-nest-failed-cant-already-there = There is already someone nested there!
+rmc-xeno-nest-failed-dead = This host is dead.
```

Comment:
```suggestion
rmc-xeno-nest-failed-dead = This host is dead, why would we want to nest it?
```

Reply:
The original version is better than this imo, or atleast a we *can not* nest it would be better. There is a very obvious reason for why they would want to nest it (so the Larva doesn't die... if you nest a dead host that falls off the wall, their sister larva will still be born). "Why would we want to do this?" feels fairly taunt-ish in tone on behalf of the game alongside the knowledge that nests deliberately allow larva to be born from dead hosts with the incubation speed increase.